embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/event/EmbraceEventService.kt

```diff
@@ -234,7 +234,9 @@ internal class EmbraceEventService(
                 sessionProperties
             )
             if (isStartupEvent(name)) {
-                logStartupSpan()
+                if (!late) {
```

Comment:
The end moment can be ended twice if the lateness timer is invoked before the actual ending. Tracing doesn't have analogue of ending spans like this yet, and even if we do, we wouldn't do it in such a way that it causes two things to be logged. 

For now, we'll only log a startup moment span if it's not "late". When we more systematically log moments as traces, we'll formalize this.